spec/lib/overrides_spec.rb

```diff
@@ -6,12 +6,19 @@
 # as the expected. If this test fails, it means that the overriden
 # file should be updated to match any change/bug fix introduced in the core
 checksums = [
+  {
+    package: "decidim-comments",
+    files: {
+      "/app/models/decidim/comments/comment.rb" => "ae74fd2277529f95aa2e6a2dd713ee7f"
+    }
+  },
```

Comment:
The checksum for this file is not the correct one as the test shows. After this change we can finally approve and deploy :)